go.mod

```diff
@@ -5,5 +5,5 @@ go 1.13
 require (
 	github.com/PuerkitoBio/goquery v1.5.0
 	github.com/aws/aws-sdk-go-v2 v0.12.0
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.7.0
```

Comment:
older version of testify brought vulnerable version of yaml. Updating to latest testify should remove vulnerable version.